apps/web/src/state/farmsV4/state/accountPositions/hooks/useAccountPositionDetailByPool.ts

```diff
@@ -0,0 +1,76 @@
+import { Protocol } from '@pancakeswap/farms'
+import { LegacyRouter } from '@pancakeswap/smart-router/legacy-router'
+import { useQuery, UseQueryResult } from '@tanstack/react-query'
+import { useCallback, useMemo } from 'react'
+import { Address, isAddressEqual } from 'viem'
+import { PoolInfo } from '../../type'
+import { getAccountV2LpDetails, getStablePairDetails } from '../fetcher'
+import { getAccountV3Positions } from '../fetcher/v3'
+import { PositionDetail, StableLPDetail, V2LPDetail } from '../type'
+
+type PoolPositionDetail = {
+  [Protocol.STABLE]: StableLPDetail
+  [Protocol.V2]: V2LPDetail
+  [Protocol.V3]: PositionDetail[]
```

Comment:
why v3 type is array?

Reply:
one V3 pool can have multi position per account